addons/web/static/src/core/hotkeys/hotkey_service.js

```diff
@@ -183,8 +183,11 @@ export const hotkeyService = {
             for (const el of getVisibleElements(activeElement, "[data-hotkey]:not(:disabled)")) {
                 const hotkey = el.dataset.hotkey;
                 const overlay = document.createElement("div");
-                overlay.className = "o_web_hotkey_overlay";
-                overlay.appendChild(document.createTextNode(hotkey.toUpperCase()));
+                overlay.className = "o_web_hotkey_overlay position-absolute top-0 bottom-0 start-0 end-0 d-flex justify-content-center align-items-center m-0 bg-black-50 h6";
```

Comment:
This line is really long and not that readable anymore (yes, the downside of utility classes...).

Maybe add them using `.classList.add("o_web_hotkey_overlay", "position-absolute"...)` instead of `.className = "..."` and let the tooling (aka Prettier) reformat it (spoiler: every class will have its own line)...

```suggestion
                overlay.classList.add(
                    "o_web_hotkey_overlay",
                    "position-absolute",
                    "top-0",
                    "bottom-0",
                    "start-0",
                    "end-0",
                    "d-flex",
                    "justify-content-center",
                    "align-items-center",
                    "m-0",
                    "bg-black-50",
                    "h6",
                );
```